.github/scripts/akto-cicd.sh

```diff
@@ -55,11 +59,28 @@ while true; do
     fi
     break
   elif [[ "$state" == "STOPPED" ]]; then
+    publishGithubCommentResponse = $(curl -s "$AKTO_DASHBOARD_URL/api/publishGithubComments" \
+          --header 'content-type: application/json' \
+          --header "X-API-KEY: $AKTO_API_KEY" \
+          --data "{
+              \"testingRunSummaryHexId\": \"$testingRunSummaryhexId\"
+          }")
     echo "Test stopped" >> $GITHUB_STEP_SUMMARY
     exit 1
     break
   else
     echo "Waiting for akto test to be completed..."
     sleep 5  # Adjust the polling interval as needed
   fi
+
+  if ((elapsed_time >= MAX_POLL_INTERVAL)); then
+    publishGithubCommentResponse = $(curl -s "$AKTO_DASHBOARD_URL/api/publishGithubComments" \
```

Comment:
Where is this variable being used? `publishGithubCommentResponse` ?

Reply:
its not, removed this